h5py/h5d.pyx

```diff
@@ -395,3 +395,38 @@ cdef class DatasetID(ObjectID):
             """
             H5Drefresh(self.id)
 
+
+    IF HDF5_VERSION >= (1, 8, 11):
+
+        def write_direct_chunk(self, offsets, bytes data, H5Z_filter_t filter_mask=H5Z_FILTER_NONE, PropID dxpl=None):
+            """ no return
```

Comment:
Please reproduce the full signature on the first line... see other functions in this module for reference.  This is needed as Cython produces a generic `f(*args, **kwds)` signature.

Also, it seems nitpicky but please break lines at 79 characters for PEP8 compliance.